File: src/main/java/com/Khata/Khata/Entity/Gig.java
package com.Khata.Khata.Entity;

import com.Khata.Khata.Enums.Days;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Entity
@Table(name = "Gigs")
public class Gig {
    @Id
    @GeneratedValue
    private Integer id;
    private Integer userId;
    private Days[] days;
    private double payrate;
    private double timePerDay;
}

File: classtracker/src/main/kotlin/com/classtrackeer/classtracker/Model/Gig.kt
package com.classtrackeer.classtracker.Model

import jakarta.persistence.Entity
import jakarta.persistence.GeneratedValue
import jakarta.persistence.Id
import jakarta.persistence.Table

@Entity
@Table(name = "Gig")
data class Gig(
    @Id
    @GeneratedValue
    val id : Int,
    val name : String,
    val payRate : Float,
    val days : Array<Day>,
    val userId : Int
) {

}